src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/Api/Methodologies/MethodologyController.cs

```diff
@@ -60,5 +59,13 @@ public async Task<ActionResult<MethodologySummaryViewModel>> GetMethodologySumma
                 .UpdateMethodologyAsync(methodologyId, request)
                 .HandleFailuresOrOk();
         }
+
+        [HttpGet("api/me/methodologies")]
+        public async Task<ActionResult<List<MethodologyPublicationsViewModel>>> GetMyMethodologyList()
+        {
+            return await _methodologyService
+                .ListWithPublicationsAsync()
+                .HandleFailuresOr(Ok);
```

Comment:
Would use `HandleFailuresOrOk` instead as it's more strictly typed than `HandleFailuresOr`. The key difference is that `HandleFailuresOrOk` will return a `ActionResult<T>`, whilst `HandleFailures` returns just `ActionResult`.

This means that we could change the return type on the controller method to say `Task<ActionResult<string>>` and it would still compile. Of course, this isn't correct and we want all of our types to align correctly :smile: